src/components/filter-box/extended-filter-box.tsx

```diff
@@ -4,6 +4,16 @@ import ReactFilterBox from "@netdata/react-filter-box"
 // So we extend base component and update its parser manually
 // to enable autosuggestions for dynamic data
 export class ExtendedFilterBox extends ReactFilterBox {
+  componentDidMount() {
+    if (super.componentDidMount) {
```

Comment:
Which is the case that we don't have a super instance?

Reply:
Correct @apostolidhs ! I think author was defensive here. Lets remove this check

Reply:
Since this is prototypal inheritance, we need to check that filter-box component has a `componentDidMount`. Otherwise there is a chance on a patch upgrade of the library that removes its own `componentDidMount` to break our code.